dds/DCPS/RTPS/RtpsCore.idl

```diff
@@ -556,11 +577,14 @@ module OpenDDS {
     };
 
     struct ParticipantProxy_t {
+      DDS::DomainId_t domainId;
+      string domainTag;
```

Comment:
Should we handle domainTag in ParameterListConverter & the logic for matching a remote participant?
https://issues.omg.org/issues/lists/dds-rtf5#issue-45519

Reply:
we will defer domainTag